tests/test_combine.py

```diff
@@ -0,0 +1,95 @@
+"""Test COMBINE archive"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Similar lines in 3 files](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4966219)